src/pages/Home/index.js

```diff
@@ -117,7 +125,7 @@ const Home = () => {
 
               <label htmlFor="topic_id">{t('home.create.topic')}</label>
               <ComboBox register={register} name="topic_id" dataSource={topicsList} />
-
+              {isTargetCreationLimit ? <p>{t('target.create.alert.limit')}</p> : null}
               <Button type="submit">{t('home.create.saveTarget')}</Button>
```

Comment:
Here you could also use `isTargetCreationLimit` to disable the button when it's true to prevent calling onSubmit unnecessarily.